services/base-service/spa/client/public/elements/fin-app.tpl.js

```diff
@@ -172,6 +172,7 @@ return html`
       <svg slot="custom-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M224 256A128 128 0 1 0 224 0a128 128 0 1 0 0 256zm-45.7 48C79.8 304 0 383.8 0 482.3C0 498.7 13.3 512 29.7 512H418.3c16.4 0 29.7-13.3 29.7-29.7C448 383.8 368.2 304 269.7 304H178.3z"/></svg>
       <a href="/${this.expertId}">Profile</a>
       <a href="/faq">Help</a>
+      <a href="/auth/login">Log In</a>
```

Comment:
@UcDust I added the login here, but it needs some TLC, like don't show login when you are logged in, and don't show logout when you are logged out, and the CSS is manked.

Reply:
@qjhart I think it should be working now. 
These links are a little strange style wise, only the first 3 links are different styles in desktop mode. But that's a brand design thing, nothing I customized for AE. 
The login/out links should hide/show now successfully at least. Let me know if anything's not quite right!

Reply:
@wrenaria this is really a UI design issue that needs approval.  With a public access, moved the login to under the profile